.gitignore

```diff
@@ -71,3 +71,5 @@ Thumbs.db
 *.exe
 *.out
 *.app
+
+*.unity
```

Comment:
I don't think that we can do this, as the scene files are .unity files and we need those in git !fix